test/unit/lib/plugins/aws/deploy/index.test.js

```diff
@@ -294,17 +294,26 @@ describe('AwsDeploy', () => {
 
     describe('"aws:deploy:finalize:cleanup" hook', () => {
       let cleanupS3BucketStub;
+      let checkIfEcrRepositoryExistsStub;
+      let cleanupEcrRepositoryStub;
       let spawnAwsCommonCleanupTempDirStub;
 
+      // TODO: Refactor to runServerless
```

Comment:
What is your opinion on refactoring this whole test suite to runServerless as a part of this Pr? I feel like it deserves another PR but on the other hand, it's pretty hard to write a meaningful test otherwise

Reply:
It's a very good call. I believe in all cases where we're about to extend/patch existing functionality, it's best first to ensure it has tests configured via `runServerless`

And, yes it's a matter for totally separate PR